instrumentation/net/http/otelhttp/internal/semconv/httpconv_benchmark_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package semconv
+
+import (
+	"net/http"
+	"net/url"
+	"testing"
+
+	"go.opentelemetry.io/otel/attribute"
+)
+
+var benchHTTPServerRequestResults []attribute.KeyValue
+
+// BenchmarkHTTPServerRequest allows comparison between different version of the HTTP server.
+// To use an alternative start this test with OTEL_HTTP_CLIENT_COMPATIBILITY_MODE set to the
+// version under test.
```

Comment:
Cannot we simply have a table-driven benchmark for different values of `OTEL_HTTP_CLIENT_COMPATIBILITY_MODE`?

Reply:
We can.  The reason not to is when you want to compare one version to a different version the benchmark needs to have the same name for `benchstat` to compare them.  

This is why I put the same benchmark in the `semconvutil`.